apps/schema/src/main.js

```diff
@@ -1,14 +1,14 @@
 import Vue from "vue";
-import App from "./App";
+import App from "./App.vue";
 import VueRouter from "vue-router";
-import SchemaSimple from "./components/SchemaSimple";
+import SchemaApp from "./components/SchemaApp.vue";
 
```

Comment:
add style sheet import for molgenis-components

Reply:
can you please explain?

Reply:
okay, required also some changes to molgenis-components to work with vite properly. But managed.